### DISCUSSION TITLE
The correct way to include styles

### DISCUSSION THREAD

**Discussion by CallOBock** [gpbl/react-day-picker#1417] [Support] [answered]
Hello. I have problem with importing styles. I'm using DayPicker v8, React v17, styled-component v5, typescript, webpack
Import: 
```
import { DayPicker } from 'react-day-picker';
import * as css from 'react-day-picker/dist/style..css';
import { format } from 'date-fns';
```
Using: 
```
<Grid.Column width={4}>
      <DayPicker  mode="single" />
 </Grid.Column>`
```
![image](https://user-images.githubusercontent.com/33382201/160597951-5aad281b-397a-4c3c-b523-6b1f81e6c31f.png)
Could you help me figure it out?

**Comment by gpbl** [comment, marked as answer]
@CallOBock your import syntax looks wrong – either you import

```
import 'react-day-picker/dist/style.css';
```

or if you use CSS modules,

```
import classNames from 'react-day-picker/dist/style.module.css';
<DayPicker classNames={{classNames}}} />
```
